docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -0,0 +1,112 @@
+
+## Installation
+
+The router package `openziti-router` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-router
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping
+
+You can opt-in to generating a configuration file and enrolling the router automatically. You will be prompted for the most relevant configuration values when installing interactively unless the answer is already known.
+
+Bootstrapping consists of these steps:
+
+1. Generate a one-shot configuration file from env answers and defaults.
+1. Enroll the router with the controller.
+
+At a minimum, you must set these to opt-in to bootstrapping.
+
+1. In `/opt/openziti/etc/router/service.env` (the answer file for the service)
+    1. Set `ZITI_BOOTSTRAP=true`
+1. In `/opt/openziti/etc/router/bootstrap.env` (the answer file for bootstrapping the router)
+    1. Set `ZITI_ENROLL_TOKEN` to the resulting token (JWT)
+
+Additionally, you probably want to set these:
+
+1. In `/opt/openziti/etc/router/bootstrap.env` (the answer file for bootstrapping the router)
+    1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller (default: localhost).
+    1. Set `ZITI_ROUTER_ADVERTISED_ADDRESS` to the permanent FQDN of the router (default: localhost). This value can not be changed after enrollment.
+    1. Set `ZITI_ROUTER_MODE` (default: none) if this router's built-in tunneler will provide a proxy for services. Mode `tproxy` requires additional kernel capabilities in `/etc/systemd/system/ziti-router.service.d/override.conf` and DNS resolver configuration for the host.
+
+### Bootstrapping the Configuration
+
+The router's configuration is loaded from a YAML file ([reference](/reference/30-configuration/router.md)). The Linux system service will generate a valid configuration during the first startup, unless one already exists. You can modify the generated configuration in `/var/lib/ziti-router/config.yml`.
+
+The filename of the configuration file, relative to the router's working directory, is given as an argument, e.g., `entrypoint.bash run config.yml` in `/lib/systemd/system/ziti-router.service` as an argument to the `ExecStart` command. You may override this in `/etc/systemd/system/ziti-router.service.d/override.conf`.
+
+Explore configuration option variables by running `ziti create config environment --help`. Any of these may be set in `/opt/openziti/etc/router/bootstrap.env` to influence the generated configuration.
+
+Disable generating a configuration by setting `ZITI_BOOTSTRAP_CONFIG=false` in `/opt/openziti/etc/router/service.env`.
+
+### Bootstrapping the Enrollment
+
+You must provide an enrollment token. Obtain the token by administratively creating the router.
```

Comment:
This is really just "enrolling the router" imo. it's really not bootstrapping anything. There's no other way to create a router than 'administratively'. Let's remove that wording and it's a guide. I think less-formal is better here. This sentence sounds more formal/rigid to me